src/main/java/org/example/homework04/Main.java

```diff
@@ -0,0 +1,49 @@
+package org.example.homework04;
+
+import org.example.homework04.domain.repository.RoleRepository;
+import org.example.homework04.domain.repository.UserRepository;
+import org.example.homework04.entity.Role;
+import org.example.homework04.entity.User;
+import org.example.homework04.repository.RoleRepositoryImpl;
+import org.example.homework04.repository.UserRepositoryImpl;
+
+public class Main {
+    public static void main(String[] args) {
+
+        RoleRepository roleRepository = new RoleRepositoryImpl();
+
+        // Create new Role.
+        Role role = roleRepository.create(new Role(null, "Root"));
```

Comment:
You can have constructor without `id`:
```suggestion
        Role role = roleRepository.create(new Role("Root"));
```